pkg/resources/masking_policy_acceptance_test.go

```diff
@@ -118,3 +118,57 @@ func maskingPolicyConfigMultiline(n string, name string, databaseName string, sc
 	}
 	`, name, databaseName, schemaName)
 }
+
+func TestAcc_MaskingPolicyMultiColumns(t *testing.T) {
+	accName := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
+
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		PreCheck:     func() { acc.TestAccPreCheck(t) },
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
```

Comment:
Could you add
```go
ConfigPlanChecks: resource.ConfigPlanChecks{
	PostApplyPostRefresh: []plancheck.PlanCheck{
		plancheck.ExpectEmptyPlan(),
	},
},
```
I think the plan is checked implicitly in the testing framework, but please add it, just to be explicit and sure that no plan is produced after the initial apply.
Other than that everything looks good. Thanks for the contribution! :)

Reply:
@sfc-gh-jcieslak 
Thanks for the review!
I added :)
https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/2686/commits/4d8a779af7c3f8ca59c3b4dc205f0724fe26bf6a

Reply:
Thank you :)